File: eel_v/web/backend/table_instance.py
import eel
import gc

from web.backend.local.dbOperations import DataBase
from mysql.connector.errors import Error

class Instance:#для выовда селекта

    number = []

    def __init__(self, db : DataBase, command):
        Instance.db = db
        Instance.result = db.make_command(command, False)
        Instance.number.append(self)
        print(Instance.__del_garbage(self))

    @eel.expose
    def show_r():
        if Error in type(Instance.result).__bases__:
            eel.show_error()
            return False
        else:
            eel.load_tables_inst(Instance.result)
    
    @eel.expose
    def give_result():
        return Instance.result

File: eel_v/web/backend/javascript/table_instance.js
eel.show_r()

eel.expose(load_tables_inst)
function load_tables_inst(result){

    let table = document.querySelector('table')

    for (let i = 0; i < result.length; i++){
        let tr = document.createElement('tr')
        tr.id = 'row'

        for (let j = 0; j < result[i].length; j++){
            let td = document.createElement('td')
            let em = document.createElement('em')
            em.innerHTML = result[i][j]

            td.appendChild(em)
            tr.appendChild(td)

        }
        table.appendChild(tr)
    }
}

eel.expose(show_error)
function show_error(){
    let table = document.querySelector('table')
    let tag = document.createElement('h2')
    tag.textContent = "Ошибка при загрузке таблицы!"
    table.appendChild(tag)
}